controllers/etcdpeer_controller.go

```diff
@@ -307,16 +312,42 @@ func (r *EtcdPeerReconciler) maybeCreatePvc(ctx context.Context, peer *etcdv1alp
 	return true, nil
 }
 
-func (r *EtcdPeerReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
+func (r *EtcdPeerReconciler) updateStatusVersion(ctx context.Context, log logr.Logger, peer *etcdv1alpha1.EtcdPeer) {
+	peer.Status.ServerVersion = ""
+	etcdConfig := etcdclient.Config{
+		Endpoints:               []string{advertiseURL(*peer, etcdPeerPort).String()},
+		Transport:               etcdclient.DefaultTransport,
+		HeaderTimeoutPerRequest: time.Second * 1,
+	}
+	etcdAPI, err := r.Etcd.New(etcdConfig)
+	if err != nil {
+		log.Error(err, "Failed to connect to ETCD")
+		return
+	}
+	etcdVersion, err := etcdAPI.GetVersion(ctx)
+	if err != nil {
+		log.Error(err, "Failed to get Etcd version")
+		return
+	}
+	log.Info("found version", "server", etcdVersion.Server)
+	peer.Status.ServerVersion = etcdVersion.Server
+}
+
+func (r *EtcdPeerReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, reterr error) {
 	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
 	defer cancel()
 
 	log := r.Log.WithValues("peer", req.NamespacedName)
 
+	// Always requeue after ten seconds, as we don't watch on the membership list. So we don't auto-detect changes made
+	// to the etcd membership API.
+	// TODO(#76) Implement custom watch on etcd membership API, and remove this `requeueAfter`
+	result := ctrl.Result{RequeueAfter: time.Second * 2}
```

Comment:
Comment says ten, but code says two. Also won't this result in the operator spinning a *lot*?

Reply:
I fixed that...I hadn't got the `triggerReconcile` function working properly in the previous version so I want'ed this short requeue to get the Kubebuilder tests to pass.